app/views/webhooks/_form.html.erb

```diff
@@ -29,5 +29,9 @@
     size: 'col-md-8', rows: 10,
     placeholder: _("Optional CAs in PEM format concatenated to verify the receiver's SSL certificate.") %>
 
+  <%= textarea_f f, :http_headers, label: _('Optional HTTP headers as JSON (ERB allowed)'),
```

Comment:
ERB is not actually allowed...

When I try to use ERB, the webhook cannot be saved due to validation of `http_headers` attribute. Before `save` it tries to `JSON.parse` a string with ERB (e.g. "<%= { \"id\": 1 } -%>") and fails with `unexpected token`.

But I guess it would be nice to support ERB here and have something similar to the editor, so users could have one macro that adds headers as key: value. And I guess this can be dropped from this PR, since I've started migration of the pages to React (and it will be my problem :D).